sources/CLSTREAMS

```diff
@@ -1,104 +1,94 @@
 (DEFINE-FILE-INFO PACKAGE "INTERLISP" READTABLE "XCL" BASE 10)
-(FILECREATED " 3-Apr-91 15:11:53" |{PELE:MV:ENVOS}<LISPCORE>SOURCES>CLSTREAMS.;4| 54013  
 
-      |changes| |to:|  (FUNCTIONS CL:WITH-INPUT-FROM-STRING)
+(FILECREATED "27-Nov-2021 13:30:46" 
```

Comment:
can't tell what changed in CLSTREAMS (besides re-pretty-printing with a different fontset.